charts/falco/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: falco
-version: 4.8.3
-appVersion: "0.38.2"
+version: 4.9.0
+appVersion: "0.39.0-rc1"
```

Comment:
```suggestion
appVersion: "0.39.0"
```

Reply:
We are going to use the RC versions until Falco 0.39.0 is out.